### PR TITLE
Added readme 

### PR DESCRIPTION
**Pull request by sarthakpant772**
I saw this project was not having a readme.md. So i tried to make it. Please insure that the steps that i have put are correct wrt this project.

### CODE REVIEW

**Comment by abhinavsri360** [comment]
@sarthakpant772 
Make sure to make PR to the `master` branch, not to the `build2.0` branch.

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]
@sarthakpant772 
Make sure to make PR to the `master` branch, not to the `build2.0` branch.

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
-[![Hits](https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fdsc-jiit-128%2FDSC-site&count_bg=%233D58C8&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=smash&edge_flat=false)](https://hits.seeyoufarm.com)
+
+![Logo](https://media-exp1.licdn.com/dms/image/C4E1BAQEeEW6C2U7kNg/company-background_10000/0/1570735746624?e=2159024400&v=beta&t=gZJK_A_8YqAW9SoU9FLkheaz8_E0Is9A2uk0cYEqdNI)
```
Comment (abhinavsri360) [inline README.md:2]: Center align the logo.

File: README.md [hunk 2]
```diff
@@ -1 +1,57 @@
-[![Hits](https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fdsc-jiit-128%2FDSC-site&count_bg=%233D58C8&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=smash&edge_flat=false)](https://hits.seeyoufarm.com)
+
+![Logo](https://media-exp1.licdn.com/dms/image/C4E1BAQEeEW6C2U7kNg/company-background_10000/0/1570735746624?e=2159024400&v=beta&t=gZJK_A_8YqAW9SoU9FLkheaz8_E0Is9A2uk0cYEqdNI)
+
+    
+# DSC-site
+
+DSC(Developers Student Club) - JIIT128 is a Google Developers program aimed for college students to help them build their mobile, web, machine learning, cloud skills, etc
```
Comment (abhinavsri360) [inline README.md:7]: Center align this part as well.

File: README.md [hunk 3]
```diff
@@ -1 +1,57 @@
...
+    npm i react
+```
+
+Making of react app
+
+```
+    npx create-react-app <appname>  //  eg:  npx create-react-app blog
+    cd <appname>  // cd blog
+    npm start
+```
```
Comment (abhinavsri360) [inline README.md:35]: I don't think we require this here

File: README.md [hunk 4]
```diff
@@ -1 +1,57 @@
...
+Clone the project
+
+```bash
+  git clone https://github.com/dsc-jiit-128/DSC-site.git
+```
+
+Go to the project directory
+
+```bash
+  cd DSC-site
```
Comment (abhinavsri360) [inline README.md:50]:
```
cd DSC-site
yarn
yarn start
```

File: README.md [hunk 5]
```diff
@@ -1 +1,57 @@
...
+```
+
+  
+## 🚀 About Us
+We are basically a community with connections all over the world. Open to any student, ranging from novice developers who are just starting, to advanced developers who want to further enhance their skills, we provide a platform for students to learn and collaborate in solving problems around them with the help of technology.
```
Comment (abhinavsri360) [inline README.md:55]: DSC(Developers Student Club) - JIIT128 is a Google Developers program aimed at college students to help them build their mobile, web, machine learning, cloud skills, etc. We are basically a community with connections all over the world. Open to any student, ranging from novice developers who are just starting, to advanced developers who want to further enhance their skills, we provide a platform for students to learn and collaborate in solving problems around them with the help of technology.

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sarthakpant772) [reply README.md:2]: ok sir cool

File: README.md (hunk 5 above)
Comment (sarthakpant772) [reply README.md:55]: ok sir

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]
Let's update the logo to this: https://drive.google.com/file/d/1S0cxxnFmQ99ExpKxuC5i3ROT2ppA0gs0/view?usp=sharing
Rest seems good to me.

**Review by abhinavsri360 — APPROVED** [review]
Looks good to me.
Great work @sarthakpant772 🎉

**Pull request merged**